appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/repositories/DiscoveryAppsRemote.kt

```diff
@@ -28,30 +29,28 @@ class Network {
         httpClient = httpClientBuilder.build()
     }
 
-    private fun getRequest(url: String): Request = Request.Builder().url(url).build()
-
-    fun getDiscoveryApps() {
-        httpClient.newCall(getRequest(GET_DISCOVERY_APPS_STAGE_URL)).enqueue(object : Callback {
+    fun getDiscoveryAppsServerData(callback: OperationResultCallback<EcosystemAppResponse>) {
+        httpClient.newCall(getRequest(GET_DISCOVERY_APPS_PROD_URL)).enqueue(object : Callback {
             override fun onFailure(call: Call, e: IOException) {
-                //TODO
-                Log.d("", "response failed")
+                callback.onError("server failed get discoveryApps ${e.message}")
             }
 
             override fun onResponse(call: Call, response: Response) {
                 if (response.isSuccessful) {
                     try {
                         val responseData = gson.fromJson(response.body()?.charStream(), EcosystemAppResponse::class.java)
-                        Log.d("", "####response version  ${responseData.version}")
-                        Log.d("", "##### app size ${responseData.apps?.size} app  ${responseData.apps?.get(0)}")
+                        callback.onResult(responseData)
```

Comment:
check body and check the char stream why is that.

Reply:
changed to string